manifests/a/Amazon/SessionManagerPlugin/1.2.279.0/Amazon.SessionManagerPlugin.locale.en-US.yaml

```
@@ -0,0 +1,19 @@
+# Created using wingetcreate 0.4.4.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: Amazon.SessionManagerPlugin
+PackageVersion: 1.2.279.0
+PackageLocale: en-US
+Publisher: AWS
+PublisherUrl: https://github.com/aws/session-manager-plugin
+PackageName: session-manager-plugin
+License: Apache 2.0 license
+LicenseUrl: https://github.com/aws/session-manager-plugin/blob/1.2.279.0/LICENSE
+ShortDescription: This plugin helps you to use the AWS Command Line Interface (AWS CLI) to start and end sessions to your managed instances.
+Tags:
+- AWS
```

```suggestion
- aws
```
This should be in lower-case.